### PR TITLE
Increase timeout for checking for empty streams

### PR DESCRIPTION
**Pull request by lemccomb**
When reading large payload files, the check in Stream.IsNullOrEmpty sometimes times out causing an empty payload error before it can complete the read operation. This change increases the timeout for those checks that would result in a Fail result from 100ms to 10 seconds.

### CODE REVIEW

File: CoseSign1.Tests/StreamExtensionsTests.cs [hunk 1]
```diff
@@ -60,4 +65,38 @@
...
+    }
+
+    public class DelayedMemoryStream : MemoryStream
+    {
+        private readonly int Delay;
+
+        public DelayedMemoryStream(byte[] content, int delay)
+        {
+            Write(content, 0, content.Length);
+            Position = 0;
```
[the remark this answers is not in the document]
Comment (lemccomb) [reply CoseSign1.Tests/StreamExtensionsTests.cs:89]: necessary for test purposes

File: CoseSign1.Tests/StreamExtensionsTests.cs [hunk 2]
```diff
@@ -60,4 +65,38 @@
...
+        memory.IsNullOrEmpty(2000).Should().Be(false);
+    }
+
+    public class DelayedMemoryStream : MemoryStream
+    {
+        private readonly int Delay;
+
+        public DelayedMemoryStream(byte[] content, int delay)
+        {
+            Write(content, 0, content.Length);
```
[the remark this answers is not in the document]
Comment (lemccomb) [reply CoseSign1.Tests/StreamExtensionsTests.cs:88]: necessary for test purposes

File: CoseSignTool/CoseCommand.cs [hunk 3]
```diff
@@ -136,7 +136,7 @@ protected ExitCode TryGetStreamFromPipeOrFile(FileInfo? file, string optionName,
             {
                 content = UseAdvancedStreamHandling ? file.GetStreamResilient(MaxWaitTime) : file.GetStreamBasic(MaxWaitTime);
                 return
-                    content.IsNullOrEmpty() ?
+                    content.IsNullOrEmpty(10000) ?
```
Comment (JosephSchutzMSFT) [inline CoseSignTool/CoseCommand.cs:139]: If we default to 10seconds, we should have an argument to downgrade this for external customers who might have empty files on low stress systems.

**Review by JosephSchutzMSFT — CHANGES_REQUESTED** [review]

File: CoseSignTool/CoseCommand.cs (hunk 3 above)
Comment (lemccomb) [reply CoseSignTool/CoseCommand.cs:139]: Adding a "FailFast" switch to account for this

**Review by JosephSchutzMSFT — APPROVED** [review]

**Pull request merged**